frontend/src/components/layouts/SettingsLayout.tsx

```diff
@@ -98,6 +98,7 @@ export default function SettingsLayout() {
       <div className="flex flex-col space-y-8 lg:flex-row lg:space-x-4 lg:space-y-0">
         <aside className="flex flex-col justify-between lg:w-1/5 max-h-screen">
           <nav className="flex flex-wrap lg:flex-col lg:space-y-1">
+            <MenuItem to="/settings/about">About</MenuItem>
```

Comment:
It's weird this is above General but General is selected by default. Can it be moved under General?